### PR TITLE
Extend LoRaBasicsStationModule capabilities to support client auth and CUPS

### PR DESCRIPTION
**Pull request by danigian**
# PR for issue #994 

## What is being addressed

Extending LoRaBasicsStationModule capabilities to support client auth certificate imports and CUPS

### CODE REVIEW

File: LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```
Comment (Mandur) [inline LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh:1]: I guess you also have to copy this file in the basicstation dockerfile?

File: LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        fi
+    else
+        cp -v $1 tc.trust
+    fi
+}
+
+tcCertCopy() {
+    if [[ "$TC_URI" == *"wss"* ]]; then
+        echo "A secure protocol was specified for LNS endpoint. Copying over certificate files".
+        conditionalCopy "$TC_TRUST_PATH" "$DEFAULT_TC_TRUST_PATH"
```
Comment (Mandur) [inline LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh:28]: Should we check as this point if TC_TRUST_PATH etc... is set and error out?

Comment (Mandur) [reply LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh:28]: Oh just saw you do it in the conditional copy, even smarter

**Review by Mandur — APPROVED** [review]

File: LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh (hunk 1 above)
Comment (danigian) [reply LoRaEngine/modules/LoRaBasicsStationModule/helper-functions.sh:1]: Done, thanks!

**Pull request merged**